bucket/gimp.json

```diff
@@ -4,7 +4,7 @@
     "homepage": "https://www.gimp.org",
     "license": "GPL-3.0-only",
     "url": "https://download.gimp.org/mirror/pub/gimp/v2.10/windows/gimp-2.10.28-setup.exe",
-    "hash": "5e9eabe5739523a9fc347b4614d919418f3335e7aab082a65f71705421e85e04",
+    "hash": "2c2e081ce541682be1abdd8bc6df13768ad9482d68000b4a7a60c764d6cec74e",
```

Comment:
I cannot comment on the correct line, but line 2 also needs an update as it's actually downloading version 2.10.28, not 2.10.24-3

Reply:
Let's handle that in a new PR for now